src/cpu/yuricat/main.cc

```diff
@@ -0,0 +1,57 @@
+#include <iostream>
+
+#include <gflags/gflags.h>
+#include <glog/logging.h>
+
+#include "base/base.h"
+#include "core/algorithm/plan.h"
+#include "core/client/ai/ai.h"
+#include "core/core_field.h"
+#include "core/frame_request.h"
+
+class SampleAI : public AI {
+public:
+    SampleAI(int argc, char* argv[]) : AI(argc, argv, "sample") {}
+    ~SampleAI() override {}
+
+    DropDecision think(int frameId, const CoreField& f, const KumipuyoSeq& seq,
+                       const PlayerState& me, const PlayerState& enemy, bool fast) const override
+    {
+        UNUSED_VARIABLE(frameId);
+        UNUSED_VARIABLE(me);
+        UNUSED_VARIABLE(enemy);
+        UNUSED_VARIABLE(fast);
+
+        LOG(INFO) << f.toDebugString() << seq.toString();
+
+        Decision best;
+        int score = -1;
+
+        Plan::iterateAvailablePlans(f, seq, 2, [&best, &score](const RefPlan& plan) {
+                int s = -9999;
+                if (plan.isRensaPlan()) {
+                    s += plan.rensaResult().chains * 10;
+                    s -= plan.decisions().size();
+                }
+
+                if (score > s) {
+					
+                    score = s;
+                    best = plan.decisions().front();
+                }
+        });
+
+		//std::cerr<<"oh"<<std::endl;
```

Comment:
Xcode settled indent. I should study how to change config of Xcode...
and I want to know what "TAB" is.